src/cmd/warpychain.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"github.com/warp-contracts/syncer/src/warpychain"
+	"github.com/warp-contracts/syncer/src/utils/logger"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(warpychainCmd)
+}
+
+var warpychainCmd = &cobra.Command{
+	Use:   "warpychain",
```

Comment:
I don't like this name :-) it suggests that we're building some custom chain :-) how about sth like`redstone-tx-syncer`